.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+  workflow_dispatch:
+
+
+jobs:
+  Build:
+    name: Build appimagetool
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - ARCH: 'x86_64'
+            docker-image: 'appimagebuild'
+          - ARCH: 'i386'
+            docker-image: 'appimagebuild-i386'
+          - ARCH: 'armhf'
+            docker-image: 'appimagebuild-armhf-cross'
+          - ARCH: 'aarch64'
+            docker-image: 'appimagebuild-aarch64-cross'
+      fail-fast: false
+    env:
+      DOCKER_IMAGE: quay.io/appimage/${{ matrix.docker-image }}
+      ARCH: ${{ matrix.ARCH }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install dependencies for ARM and AARCH64
+        if: endsWith(matrix.docker-image, 'cross') 
+        run: |
```

Comment:
There's a much easier alternative:

```
      - name: Set up QEMU integration for Docker
        run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

This works reliably with *all* image architectures.